2018-07-11 00:04:29,198 logger.py[line:23] [process:25102] MainThread INFO init log
2018-07-11 00:04:29,441 _internal.py[line:88] [process:25113] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:04:29,485 _internal.py[line:88] [process:25113] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:04:31,293 logger.py[line:23] [process:25116] MainThread INFO init log
2018-07-11 00:04:31,545 _internal.py[line:88] [process:25128] MainThread WARNING  * Debugger is active!
2018-07-11 00:04:31,583 _internal.py[line:88] [process:25128] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:04:43,934 _internal.py[line:88] [process:25128] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 00:04:43] "GET / HTTP/1.1" 200 -
2018-07-11 00:05:01,296 _server.py[line:393] [process:25112] <concurrent.futures.thread.ThreadPoolExecutor object at 0x10ff22630>_0 ERROR Exception calling application: 'GetNoteBookRequest' object has no attribute 'user_id'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/notebook/services/notebook/get_notebook_service.py", line 34, in GetNotebookInfo
    user_id = request.user_id
AttributeError: 'GetNoteBookRequest' object has no attribute 'user_id'
2018-07-11 00:05:01,335 _internal.py[line:88] [process:25128] Thread-25 INFO 127.0.0.1 - - [11/Jul/2018 00:05:01] "GET /notebook HTTP/1.1" 500 -
2018-07-11 00:05:01,366 _internal.py[line:88] [process:25128] Thread-26 INFO 127.0.0.1 - - [11/Jul/2018 00:05:01] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:05:01,370 _internal.py[line:88] [process:25128] Thread-27 INFO 127.0.0.1 - - [11/Jul/2018 00:05:01] "GET /notebook?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:05:01,372 _internal.py[line:88] [process:25128] Thread-32 INFO 127.0.0.1 - - [11/Jul/2018 00:05:01] "GET /notebook?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:05:01,412 _internal.py[line:88] [process:25128] Thread-33 INFO 127.0.0.1 - - [11/Jul/2018 00:05:01] "GET /notebook?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:05:01,509 _internal.py[line:88] [process:25128] Thread-34 INFO 127.0.0.1 - - [11/Jul/2018 00:05:01] "GET /notebook?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:05:01,600 _internal.py[line:88] [process:25128] Thread-35 INFO 127.0.0.1 - - [11/Jul/2018 00:05:01] "GET /notebook?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:05:07,314 _internal.py[line:88] [process:25128] Thread-36 INFO 127.0.0.1 - - [11/Jul/2018 00:05:07] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:06:00,513 _internal.py[line:88] [process:25128] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/notebook/notebook_service.py', reloading
2018-07-11 00:06:00,515 _internal.py[line:88] [process:25128] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/notebook/notebook_service.py', reloading
2018-07-11 00:06:47,273 logger.py[line:23] [process:25433] MainThread INFO init log
2018-07-11 00:06:47,519 _internal.py[line:88] [process:25444] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:06:47,565 _internal.py[line:88] [process:25444] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:06:49,330 logger.py[line:23] [process:25447] MainThread INFO init log
2018-07-11 00:06:49,593 _internal.py[line:88] [process:25460] MainThread WARNING  * Debugger is active!
2018-07-11 00:06:49,639 _internal.py[line:88] [process:25460] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:06:51,519 _server.py[line:393] [process:25443] <concurrent.futures.thread.ThreadPoolExecutor object at 0x10aa70630>_0 ERROR Exception calling application: 'GetNoteBookRequest' object has no attribute 'user_id'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/notebook/services/notebook/get_notebook_service.py", line 34, in GetNotebookInfo
    user_id = request.user_id
AttributeError: 'GetNoteBookRequest' object has no attribute 'user_id'
2018-07-11 00:06:51,565 _internal.py[line:88] [process:25460] Thread-16 INFO 127.0.0.1 - - [11/Jul/2018 00:06:51] "GET /notebook HTTP/1.1" 500 -
2018-07-11 00:06:51,608 _internal.py[line:88] [process:25460] Thread-19 INFO 127.0.0.1 - - [11/Jul/2018 00:06:51] "GET /notebook?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:06:51,615 _internal.py[line:88] [process:25460] Thread-18 INFO 127.0.0.1 - - [11/Jul/2018 00:06:51] "GET /notebook?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:06:51,619 _internal.py[line:88] [process:25460] Thread-17 INFO 127.0.0.1 - - [11/Jul/2018 00:06:51] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:06:51,681 _internal.py[line:88] [process:25460] Thread-20 INFO 127.0.0.1 - - [11/Jul/2018 00:06:51] "GET /notebook?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:06:51,745 _internal.py[line:88] [process:25460] Thread-25 INFO 127.0.0.1 - - [11/Jul/2018 00:06:51] "GET /notebook?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:10:18,044 logger.py[line:23] [process:25916] MainThread INFO init log
2018-07-11 00:10:18,288 _internal.py[line:88] [process:25935] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:10:18,328 _internal.py[line:88] [process:25935] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:10:20,077 logger.py[line:23] [process:25938] MainThread INFO init log
2018-07-11 00:10:20,328 _internal.py[line:88] [process:25953] MainThread WARNING  * Debugger is active!
2018-07-11 00:10:20,357 _internal.py[line:88] [process:25953] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:10:56,257 _server.py[line:393] [process:25934] <concurrent.futures.thread.ThreadPoolExecutor object at 0x1103035f8>_0 ERROR Exception calling application: 'GetNoteBookRequest' object has no attribute 'user_id'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/notebook/services/notebook/get_notebook_service.py", line 34, in GetNotebookInfo
    user_id = request.user_id
AttributeError: 'GetNoteBookRequest' object has no attribute 'user_id'
2018-07-11 00:10:56,290 _internal.py[line:88] [process:25953] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 00:10:56] "GET /notebook HTTP/1.1" 500 -
2018-07-11 00:10:56,331 _internal.py[line:88] [process:25953] Thread-22 INFO 127.0.0.1 - - [11/Jul/2018 00:10:56] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:10:56,337 _internal.py[line:88] [process:25953] Thread-24 INFO 127.0.0.1 - - [11/Jul/2018 00:10:56] "GET /notebook?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:10:56,342 _internal.py[line:88] [process:25953] Thread-23 INFO 127.0.0.1 - - [11/Jul/2018 00:10:56] "GET /notebook?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:10:56,399 _internal.py[line:88] [process:25953] Thread-25 INFO 127.0.0.1 - - [11/Jul/2018 00:10:56] "GET /notebook?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:10:56,454 _internal.py[line:88] [process:25953] Thread-30 INFO 127.0.0.1 - - [11/Jul/2018 00:10:56] "GET /notebook?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:12:52,527 _internal.py[line:88] [process:25953] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/proto/user_pb2_grpc.py', reloading
2018-07-11 00:12:52,532 _internal.py[line:88] [process:25953] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/proto/__init__.py', reloading
2018-07-11 00:12:52,533 _internal.py[line:88] [process:25953] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/proto/api_pb2_grpc.py', reloading
2018-07-11 00:12:52,535 _internal.py[line:88] [process:25953] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/proto/notebook_pb2_grpc.py', reloading
2018-07-11 00:12:52,536 _internal.py[line:88] [process:25953] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/proto/notebook_pb2.py', reloading
2018-07-11 00:12:52,538 _internal.py[line:88] [process:25953] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/proto/api_pb2.py', reloading
2018-07-11 00:12:52,539 _internal.py[line:88] [process:25953] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/proto/user_pb2.py', reloading
2018-07-11 00:13:11,770 logger.py[line:23] [process:26423] MainThread INFO init log
2018-07-11 00:13:12,013 _internal.py[line:88] [process:26440] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:13:12,054 _internal.py[line:88] [process:26440] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:13:13,713 logger.py[line:23] [process:26445] MainThread INFO init log
2018-07-11 00:13:13,950 _internal.py[line:88] [process:26458] MainThread WARNING  * Debugger is active!
2018-07-11 00:13:13,983 _internal.py[line:88] [process:26458] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:13:17,901 _server.py[line:393] [process:26439] <concurrent.futures.thread.ThreadPoolExecutor object at 0x10562c5c0>_0 ERROR Exception calling application: 'GetNoteBookRequest' object has no attribute 'user_id'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/notebook/services/notebook/get_notebook_service.py", line 34, in GetNotebookInfo
    user_id = request.user_id
AttributeError: 'GetNoteBookRequest' object has no attribute 'user_id'
2018-07-11 00:13:17,933 _internal.py[line:88] [process:26458] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 00:13:17] "GET /notebook HTTP/1.1" 500 -
2018-07-11 00:13:17,973 _internal.py[line:88] [process:26458] Thread-23 INFO 127.0.0.1 - - [11/Jul/2018 00:13:17] "GET /notebook?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:13:17,977 _internal.py[line:88] [process:26458] Thread-24 INFO 127.0.0.1 - - [11/Jul/2018 00:13:17] "GET /notebook?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:13:17,980 _internal.py[line:88] [process:26458] Thread-22 INFO 127.0.0.1 - - [11/Jul/2018 00:13:17] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:13:18,039 _internal.py[line:88] [process:26458] Thread-25 INFO 127.0.0.1 - - [11/Jul/2018 00:13:18] "GET /notebook?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:13:18,093 _internal.py[line:88] [process:26458] Thread-26 INFO 127.0.0.1 - - [11/Jul/2018 00:13:18] "GET /notebook?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:20:46,929 logger.py[line:23] [process:27193] MainThread INFO init log
2018-07-11 00:20:47,239 _internal.py[line:88] [process:27206] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:20:47,303 _internal.py[line:88] [process:27206] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:20:48,822 logger.py[line:23] [process:27209] MainThread INFO init log
2018-07-11 00:20:49,254 _internal.py[line:88] [process:27220] MainThread WARNING  * Debugger is active!
2018-07-11 00:20:49,292 _internal.py[line:88] [process:27220] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:21:10,480 _server.py[line:393] [process:27205] <concurrent.futures.thread.ThreadPoolExecutor object at 0x111fd3668>_0 ERROR Exception calling application: 'GetNoteBookRequest' object has no attribute 'user_id'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/notebook/services/notebook/get_notebook_service.py", line 34, in GetNotebookInfo
    user_id = request.user_id
AttributeError: 'GetNoteBookRequest' object has no attribute 'user_id'
2018-07-11 00:21:10,540 _internal.py[line:88] [process:27220] Thread-15 INFO 127.0.0.1 - - [11/Jul/2018 00:21:10] "GET /notebook HTTP/1.1" 500 -
2018-07-11 00:21:10,606 _internal.py[line:88] [process:27220] Thread-37 INFO 127.0.0.1 - - [11/Jul/2018 00:21:10] "GET /notebook?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:21:10,611 _internal.py[line:88] [process:27220] Thread-35 INFO 127.0.0.1 - - [11/Jul/2018 00:21:10] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:21:10,617 _internal.py[line:88] [process:27220] Thread-36 INFO 127.0.0.1 - - [11/Jul/2018 00:21:10] "GET /notebook?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:21:12,552 _server.py[line:393] [process:27205] <concurrent.futures.thread.ThreadPoolExecutor object at 0x111fd3668>_0 ERROR Exception calling application: 'GetNoteBookRequest' object has no attribute 'user_id'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/notebook/services/notebook/get_notebook_service.py", line 34, in GetNotebookInfo
    user_id = request.user_id
AttributeError: 'GetNoteBookRequest' object has no attribute 'user_id'
2018-07-11 00:21:12,707 _internal.py[line:88] [process:27220] Thread-38 INFO 127.0.0.1 - - [11/Jul/2018 00:21:12] "GET /notebook HTTP/1.1" 500 -
2018-07-11 00:21:12,753 _internal.py[line:88] [process:27220] Thread-39 INFO 127.0.0.1 - - [11/Jul/2018 00:21:12] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:21:12,759 _internal.py[line:88] [process:27220] Thread-43 INFO 127.0.0.1 - - [11/Jul/2018 00:21:12] "GET /notebook?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:21:12,766 _internal.py[line:88] [process:27220] Thread-44 INFO 127.0.0.1 - - [11/Jul/2018 00:21:12] "GET /notebook?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:21:12,824 _internal.py[line:88] [process:27220] Thread-45 INFO 127.0.0.1 - - [11/Jul/2018 00:21:12] "GET /notebook?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:21:12,875 _internal.py[line:88] [process:27220] Thread-46 INFO 127.0.0.1 - - [11/Jul/2018 00:21:12] "GET /notebook?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:27:43,216 logger.py[line:23] [process:28017] MainThread INFO init log
2018-07-11 00:27:43,576 _internal.py[line:88] [process:28040] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:27:43,654 _internal.py[line:88] [process:28040] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:27:51,727 logger.py[line:23] [process:28043] MainThread INFO init log
2018-07-11 00:27:52,034 _internal.py[line:88] [process:28076] MainThread WARNING  * Debugger is active!
2018-07-11 00:27:52,077 _internal.py[line:88] [process:28076] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:27:57,647 _server.py[line:393] [process:28039] <concurrent.futures.thread.ThreadPoolExecutor object at 0x10d3606a0>_0 ERROR Exception calling application: 'GetNoteBookRequest' object has no attribute 'user_id'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/notebook/services/notebook/get_notebook_service.py", line 34, in GetNotebookInfo
    user_id = request.user_id
AttributeError: 'GetNoteBookRequest' object has no attribute 'user_id'
2018-07-11 00:27:57,723 _internal.py[line:88] [process:28076] Thread-15 INFO 127.0.0.1 - - [11/Jul/2018 00:27:57] "GET /notebook HTTP/1.1" 500 -
2018-07-11 00:27:57,782 _internal.py[line:88] [process:28076] Thread-18 INFO 127.0.0.1 - - [11/Jul/2018 00:27:57] "GET /notebook?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:27:57,787 _internal.py[line:88] [process:28076] Thread-16 INFO 127.0.0.1 - - [11/Jul/2018 00:27:57] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:27:57,790 _internal.py[line:88] [process:28076] Thread-17 INFO 127.0.0.1 - - [11/Jul/2018 00:27:57] "GET /notebook?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:27:57,839 _internal.py[line:88] [process:28076] Thread-19 INFO 127.0.0.1 - - [11/Jul/2018 00:27:57] "GET /notebook?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:27:57,902 _internal.py[line:88] [process:28076] Thread-32 INFO 127.0.0.1 - - [11/Jul/2018 00:27:57] "GET /notebook?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:28:36,462 logger.py[line:23] [process:28208] MainThread INFO init log
2018-07-11 00:28:36,769 _internal.py[line:88] [process:28226] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:28:36,830 _internal.py[line:88] [process:28226] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:28:38,356 logger.py[line:23] [process:28229] MainThread INFO init log
2018-07-11 00:28:38,658 _internal.py[line:88] [process:28244] MainThread WARNING  * Debugger is active!
2018-07-11 00:28:38,691 _internal.py[line:88] [process:28244] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:28:45,865 _server.py[line:393] [process:28225] <concurrent.futures.thread.ThreadPoolExecutor object at 0x10549f6a0>_0 ERROR Exception calling application: 'GetNoteBookRequest' object has no attribute 'user_id'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/notebook/services/notebook/get_notebook_service.py", line 34, in GetNotebookInfo
    user_id = request.user_id
AttributeError: 'GetNoteBookRequest' object has no attribute 'user_id'
2018-07-11 00:28:45,924 _internal.py[line:88] [process:28244] Thread-16 INFO 127.0.0.1 - - [11/Jul/2018 00:28:45] "GET /notebook HTTP/1.1" 500 -
2018-07-11 00:28:45,965 _internal.py[line:88] [process:28244] Thread-18 INFO 127.0.0.1 - - [11/Jul/2018 00:28:45] "GET /notebook?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:28:45,968 _internal.py[line:88] [process:28244] Thread-19 INFO 127.0.0.1 - - [11/Jul/2018 00:28:45] "GET /notebook?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:28:45,974 _internal.py[line:88] [process:28244] Thread-17 INFO 127.0.0.1 - - [11/Jul/2018 00:28:45] "GET /notebook?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:28:46,020 _internal.py[line:88] [process:28244] Thread-20 INFO 127.0.0.1 - - [11/Jul/2018 00:28:46] "GET /notebook?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:28:46,090 _internal.py[line:88] [process:28244] Thread-32 INFO 127.0.0.1 - - [11/Jul/2018 00:28:46] "GET /notebook?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:32:13,543 _internal.py[line:88] [process:28244] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/user/service.py', reloading
2018-07-11 00:32:13,545 _internal.py[line:88] [process:28244] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/user/service.py', reloading
2018-07-11 00:35:18,708 logger.py[line:23] [process:29029] MainThread INFO init log
2018-07-11 00:35:18,954 _internal.py[line:88] [process:29043] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:35:18,994 _internal.py[line:88] [process:29043] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:35:20,669 logger.py[line:23] [process:29047] MainThread INFO init log
2018-07-11 00:35:20,898 _internal.py[line:88] [process:29060] MainThread WARNING  * Debugger is active!
2018-07-11 00:35:20,934 _internal.py[line:88] [process:29060] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:35:40,953 _internal.py[line:88] [process:29060] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/notebook/notebook_client.py', reloading
2018-07-11 00:35:40,954 _internal.py[line:88] [process:29060] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/notebook/notebook_client.py', reloading
2018-07-11 00:35:46,271 logger.py[line:23] [process:29140] MainThread INFO init log
2018-07-11 00:35:46,506 _internal.py[line:88] [process:29154] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:35:46,544 _internal.py[line:88] [process:29154] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:35:48,127 logger.py[line:23] [process:29159] MainThread INFO init log
2018-07-11 00:35:48,350 _internal.py[line:88] [process:29172] MainThread WARNING  * Debugger is active!
2018-07-11 00:35:48,386 _internal.py[line:88] [process:29172] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:35:59,510 _internal.py[line:88] [process:29172] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 00:35:59] "GET /notebook HTTP/1.1" 200 -
2018-07-11 00:36:03,209 _internal.py[line:88] [process:29172] Thread-22 INFO 127.0.0.1 - - [11/Jul/2018 00:36:03] "GET /notebook HTTP/1.1" 200 -
2018-07-11 00:36:08,310 _internal.py[line:88] [process:29172] Thread-23 INFO 127.0.0.1 - - [11/Jul/2018 00:36:08] "GET /user HTTP/1.1" 500 -
2018-07-11 00:36:08,350 _internal.py[line:88] [process:29172] Thread-25 INFO 127.0.0.1 - - [11/Jul/2018 00:36:08] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:36:08,352 _internal.py[line:88] [process:29172] Thread-24 INFO 127.0.0.1 - - [11/Jul/2018 00:36:08] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:36:08,357 _internal.py[line:88] [process:29172] Thread-26 INFO 127.0.0.1 - - [11/Jul/2018 00:36:08] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:36:08,408 _internal.py[line:88] [process:29172] Thread-31 INFO 127.0.0.1 - - [11/Jul/2018 00:36:08] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:36:08,455 _internal.py[line:88] [process:29172] Thread-32 INFO 127.0.0.1 - - [11/Jul/2018 00:36:08] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:36:08,525 _internal.py[line:88] [process:29172] Thread-33 INFO 127.0.0.1 - - [11/Jul/2018 00:36:08] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:36:34,244 _internal.py[line:88] [process:29172] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/user/user_client.py', reloading
2018-07-11 00:36:34,246 _internal.py[line:88] [process:29172] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/user/user_client.py', reloading
2018-07-11 00:36:49,840 logger.py[line:23] [process:29318] MainThread INFO init log
2018-07-11 00:36:50,079 _internal.py[line:88] [process:29332] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:36:50,125 _internal.py[line:88] [process:29332] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:36:51,749 logger.py[line:23] [process:29337] MainThread INFO init log
2018-07-11 00:36:52,122 _internal.py[line:88] [process:29352] MainThread WARNING  * Debugger is active!
2018-07-11 00:36:52,167 _internal.py[line:88] [process:29352] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:37:07,498 _internal.py[line:88] [process:29352] Thread-15 INFO 127.0.0.1 - - [11/Jul/2018 00:37:07] "GET /user HTTP/1.1" 500 -
2018-07-11 00:37:07,537 _internal.py[line:88] [process:29352] Thread-33 INFO 127.0.0.1 - - [11/Jul/2018 00:37:07] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:37:07,540 _internal.py[line:88] [process:29352] Thread-35 INFO 127.0.0.1 - - [11/Jul/2018 00:37:07] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:37:07,543 _internal.py[line:88] [process:29352] Thread-34 INFO 127.0.0.1 - - [11/Jul/2018 00:37:07] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:37:17,551 _internal.py[line:88] [process:29352] Thread-37 INFO 127.0.0.1 - - [11/Jul/2018 00:37:17] "GET /user HTTP/1.1" 500 -
2018-07-11 00:37:17,564 _internal.py[line:88] [process:29352] Thread-36 INFO 127.0.0.1 - - [11/Jul/2018 00:37:17] "GET /user HTTP/1.1" 500 -
2018-07-11 00:37:17,580 _internal.py[line:88] [process:29352] Thread-38 INFO 127.0.0.1 - - [11/Jul/2018 00:37:17] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:37:17,583 _internal.py[line:88] [process:29352] Thread-39 INFO 127.0.0.1 - - [11/Jul/2018 00:37:17] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:37:17,586 _internal.py[line:88] [process:29352] Thread-40 INFO 127.0.0.1 - - [11/Jul/2018 00:37:17] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:37:17,636 _internal.py[line:88] [process:29352] Thread-41 INFO 127.0.0.1 - - [11/Jul/2018 00:37:17] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:37:17,689 _internal.py[line:88] [process:29352] Thread-42 INFO 127.0.0.1 - - [11/Jul/2018 00:37:17] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:38:44,305 _internal.py[line:88] [process:29352] Thread-45 INFO 127.0.0.1 - - [11/Jul/2018 00:38:44] "GET /user HTTP/1.1" 500 -
2018-07-11 00:38:44,341 _internal.py[line:88] [process:29352] Thread-49 INFO 127.0.0.1 - - [11/Jul/2018 00:38:44] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:38:44,346 _internal.py[line:88] [process:29352] Thread-50 INFO 127.0.0.1 - - [11/Jul/2018 00:38:44] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:38:44,358 _internal.py[line:88] [process:29352] Thread-51 INFO 127.0.0.1 - - [11/Jul/2018 00:38:44] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:38:44,433 _internal.py[line:88] [process:29352] Thread-52 INFO 127.0.0.1 - - [11/Jul/2018 00:38:44] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:38:44,487 _internal.py[line:88] [process:29352] Thread-53 INFO 127.0.0.1 - - [11/Jul/2018 00:38:44] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:39:45,102 _internal.py[line:88] [process:29352] Thread-54 INFO 127.0.0.1 - - [11/Jul/2018 00:39:45] "GET /user HTTP/1.1" 500 -
2018-07-11 00:39:45,137 _internal.py[line:88] [process:29352] Thread-57 INFO 127.0.0.1 - - [11/Jul/2018 00:39:45] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:39:45,145 _internal.py[line:88] [process:29352] Thread-58 INFO 127.0.0.1 - - [11/Jul/2018 00:39:45] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:39:45,149 _internal.py[line:88] [process:29352] Thread-59 INFO 127.0.0.1 - - [11/Jul/2018 00:39:45] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:39:45,205 _internal.py[line:88] [process:29352] Thread-60 INFO 127.0.0.1 - - [11/Jul/2018 00:39:45] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:39:45,283 _internal.py[line:88] [process:29352] Thread-61 INFO 127.0.0.1 - - [11/Jul/2018 00:39:45] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:40:48,068 logger.py[line:23] [process:29801] MainThread INFO init log
2018-07-11 00:40:48,299 _internal.py[line:88] [process:29811] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:40:48,347 _internal.py[line:88] [process:29811] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:40:49,914 logger.py[line:23] [process:29816] MainThread INFO init log
2018-07-11 00:40:50,145 _internal.py[line:88] [process:29828] MainThread WARNING  * Debugger is active!
2018-07-11 00:40:50,181 _internal.py[line:88] [process:29828] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:41:21,978 _internal.py[line:88] [process:29828] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/user/main.py', reloading
2018-07-11 00:41:21,980 _internal.py[line:88] [process:29828] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/user/main.py', reloading
2018-07-11 00:41:26,260 logger.py[line:23] [process:29881] MainThread INFO init log
2018-07-11 00:41:26,494 _internal.py[line:88] [process:29891] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:41:26,543 _internal.py[line:88] [process:29891] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:41:28,148 logger.py[line:23] [process:29896] MainThread INFO init log
2018-07-11 00:41:28,375 _internal.py[line:88] [process:29905] MainThread WARNING  * Debugger is active!
2018-07-11 00:41:28,415 _internal.py[line:88] [process:29905] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:41:46,212 _internal.py[line:88] [process:29905] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/main.py', reloading
2018-07-11 00:41:46,214 _internal.py[line:88] [process:29905] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/main.py', reloading
2018-07-11 00:41:52,358 logger.py[line:23] [process:29973] MainThread INFO init log
2018-07-11 00:41:52,607 _internal.py[line:88] [process:29986] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:41:52,653 _internal.py[line:88] [process:29986] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:41:54,346 logger.py[line:23] [process:29989] MainThread INFO init log
2018-07-11 00:41:54,585 _internal.py[line:88] [process:30003] MainThread WARNING  * Debugger is active!
2018-07-11 00:41:54,617 _internal.py[line:88] [process:30003] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:41:59,006 _server.py[line:393] [process:29984] <concurrent.futures.thread.ThreadPoolExecutor object at 0x113e6e518>_0 ERROR Exception calling application: 'GetUserRequest' object has no attribute 'userId'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/user/services/get_user_service.py", line 34, in GetUserInfo
    user_id = request.userId
AttributeError: 'GetUserRequest' object has no attribute 'userId'
2018-07-11 00:41:59,041 _internal.py[line:88] [process:30003] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 00:41:59] "GET /user HTTP/1.1" 500 -
2018-07-11 00:41:59,084 _internal.py[line:88] [process:30003] Thread-23 INFO 127.0.0.1 - - [11/Jul/2018 00:41:59] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:41:59,088 _internal.py[line:88] [process:30003] Thread-22 INFO 127.0.0.1 - - [11/Jul/2018 00:41:59] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:41:59,095 _internal.py[line:88] [process:30003] Thread-28 INFO 127.0.0.1 - - [11/Jul/2018 00:41:59] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:41:59,159 _internal.py[line:88] [process:30003] Thread-29 INFO 127.0.0.1 - - [11/Jul/2018 00:41:59] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:41:59,219 _internal.py[line:88] [process:30003] Thread-30 INFO 127.0.0.1 - - [11/Jul/2018 00:41:59] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:42:28,100 _internal.py[line:88] [process:30003] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/user/user_client.py', reloading
2018-07-11 00:42:28,101 _internal.py[line:88] [process:30003] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/user/user_client.py', reloading
2018-07-11 00:42:34,133 logger.py[line:23] [process:30088] MainThread INFO init log
2018-07-11 00:42:34,379 _internal.py[line:88] [process:30100] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:42:34,418 _internal.py[line:88] [process:30100] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:42:36,032 logger.py[line:23] [process:30103] MainThread INFO init log
2018-07-11 00:42:36,321 _internal.py[line:88] [process:30117] MainThread WARNING  * Debugger is active!
2018-07-11 00:42:36,371 _internal.py[line:88] [process:30117] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:42:38,284 _internal.py[line:88] [process:30117] Thread-18 INFO 127.0.0.1 - - [11/Jul/2018 00:42:38] "GET /user HTTP/1.1" 500 -
2018-07-11 00:42:38,328 _internal.py[line:88] [process:30117] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 00:42:38] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:42:38,333 _internal.py[line:88] [process:30117] Thread-19 INFO 127.0.0.1 - - [11/Jul/2018 00:42:38] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:42:38,341 _internal.py[line:88] [process:30117] Thread-20 INFO 127.0.0.1 - - [11/Jul/2018 00:42:38] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:42:38,410 _internal.py[line:88] [process:30117] Thread-22 INFO 127.0.0.1 - - [11/Jul/2018 00:42:38] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:42:38,479 _internal.py[line:88] [process:30117] Thread-23 INFO 127.0.0.1 - - [11/Jul/2018 00:42:38] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:42:56,866 _internal.py[line:88] [process:30117] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/user/user_client.py', reloading
2018-07-11 00:42:56,868 _internal.py[line:88] [process:30117] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/user/user_client.py', reloading
2018-07-11 00:43:24,942 logger.py[line:23] [process:30231] MainThread INFO init log
2018-07-11 00:43:25,187 _internal.py[line:88] [process:30247] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:43:25,224 _internal.py[line:88] [process:30247] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:43:26,950 logger.py[line:23] [process:30250] MainThread INFO init log
2018-07-11 00:43:27,196 _internal.py[line:88] [process:30265] MainThread WARNING  * Debugger is active!
2018-07-11 00:43:27,234 _internal.py[line:88] [process:30265] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:43:30,376 _server.py[line:393] [process:30245] <concurrent.futures.thread.ThreadPoolExecutor object at 0x110cb8518>_0 ERROR Exception calling application: 'GetUserRequest' object has no attribute 'userId'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/user/services/get_user_service.py", line 34, in GetUserInfo
    user_id = request.userId
AttributeError: 'GetUserRequest' object has no attribute 'userId'
2018-07-11 00:43:30,419 _internal.py[line:88] [process:30265] Thread-20 INFO 127.0.0.1 - - [11/Jul/2018 00:43:30] "GET /user HTTP/1.1" 500 -
2018-07-11 00:43:30,461 _internal.py[line:88] [process:30265] Thread-23 INFO 127.0.0.1 - - [11/Jul/2018 00:43:30] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:43:30,468 _internal.py[line:88] [process:30265] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 00:43:30] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:43:30,475 _internal.py[line:88] [process:30265] Thread-22 INFO 127.0.0.1 - - [11/Jul/2018 00:43:30] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:43:30,541 _internal.py[line:88] [process:30265] Thread-24 INFO 127.0.0.1 - - [11/Jul/2018 00:43:30] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:43:30,604 _internal.py[line:88] [process:30265] Thread-29 INFO 127.0.0.1 - - [11/Jul/2018 00:43:30] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:45:10,144 _internal.py[line:88] [process:30265] Thread-31 INFO  * To enable the debugger you need to enter the security pin:
2018-07-11 00:45:10,146 _internal.py[line:88] [process:30265] Thread-31 INFO  * Debugger pin code: 202-833-239
2018-07-11 00:45:10,148 _internal.py[line:88] [process:30265] Thread-31 INFO 127.0.0.1 - - [11/Jul/2018 00:45:10] "GET /user?__debugger__=yes&cmd=printpin&s=gaviyR3sOCeXWNazBdeo HTTP/1.1" 200 -
2018-07-11 00:45:14,719 _internal.py[line:88] [process:30265] Thread-32 INFO 127.0.0.1 - - [11/Jul/2018 00:45:14] "GET /user?__debugger__=yes&cmd=pinauth&pin=&s=gaviyR3sOCeXWNazBdeo HTTP/1.1" 200 -
2018-07-11 00:47:28,885 _server.py[line:393] [process:30245] <concurrent.futures.thread.ThreadPoolExecutor object at 0x110cb8518>_0 ERROR Exception calling application: 'GetUserRequest' object has no attribute 'userId'
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/grpc/_server.py", line 385, in _call_behavior
    return behavior(argument, context), True
  File "/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/user/services/get_user_service.py", line 34, in GetUserInfo
    user_id = request.userId
AttributeError: 'GetUserRequest' object has no attribute 'userId'
2018-07-11 00:47:28,918 _internal.py[line:88] [process:30265] Thread-33 INFO 127.0.0.1 - - [11/Jul/2018 00:47:28] "GET /user HTTP/1.1" 500 -
2018-07-11 00:47:28,948 _internal.py[line:88] [process:30265] Thread-36 INFO 127.0.0.1 - - [11/Jul/2018 00:47:28] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:47:28,953 _internal.py[line:88] [process:30265] Thread-34 INFO 127.0.0.1 - - [11/Jul/2018 00:47:28] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:47:28,958 _internal.py[line:88] [process:30265] Thread-35 INFO 127.0.0.1 - - [11/Jul/2018 00:47:28] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:47:29,025 _internal.py[line:88] [process:30265] Thread-37 INFO 127.0.0.1 - - [11/Jul/2018 00:47:29] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:47:29,085 _internal.py[line:88] [process:30265] Thread-38 INFO 127.0.0.1 - - [11/Jul/2018 00:47:29] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:48:59,857 logger.py[line:23] [process:30882] MainThread INFO init log
2018-07-11 00:49:00,103 _internal.py[line:88] [process:30893] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:49:00,145 _internal.py[line:88] [process:30893] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:49:01,974 logger.py[line:23] [process:30896] MainThread INFO init log
2018-07-11 00:49:02,231 _internal.py[line:88] [process:30909] MainThread WARNING  * Debugger is active!
2018-07-11 00:49:02,266 _internal.py[line:88] [process:30909] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:49:03,241 _internal.py[line:88] [process:30909] Thread-15 INFO 127.0.0.1 - - [11/Jul/2018 00:49:03] "GET /user HTTP/1.1" 500 -
2018-07-11 00:49:03,283 _internal.py[line:88] [process:30909] Thread-18 INFO 127.0.0.1 - - [11/Jul/2018 00:49:03] "GET /user?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
2018-07-11 00:49:03,288 _internal.py[line:88] [process:30909] Thread-17 INFO 127.0.0.1 - - [11/Jul/2018 00:49:03] "GET /user?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
2018-07-11 00:49:03,295 _internal.py[line:88] [process:30909] Thread-16 INFO 127.0.0.1 - - [11/Jul/2018 00:49:03] "GET /user?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
2018-07-11 00:49:03,350 _internal.py[line:88] [process:30909] Thread-19 INFO 127.0.0.1 - - [11/Jul/2018 00:49:03] "GET /user?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1" 200 -
2018-07-11 00:49:03,408 _internal.py[line:88] [process:30909] Thread-20 INFO 127.0.0.1 - - [11/Jul/2018 00:49:03] "GET /user?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
2018-07-11 00:49:21,080 logger.py[line:23] [process:30956] MainThread INFO init log
2018-07-11 00:49:21,327 _internal.py[line:88] [process:30975] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:49:21,372 _internal.py[line:88] [process:30975] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:49:23,132 logger.py[line:23] [process:30978] MainThread INFO init log
2018-07-11 00:49:23,368 _internal.py[line:88] [process:30993] MainThread WARNING  * Debugger is active!
2018-07-11 00:49:23,392 _internal.py[line:88] [process:30993] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 00:49:23,677 _internal.py[line:88] [process:30993] Thread-14 INFO 127.0.0.1 - - [11/Jul/2018 00:49:23] "GET /user HTTP/1.1" 200 -
2018-07-11 00:50:08,963 _internal.py[line:88] [process:30993] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/user/user_client.py', reloading
2018-07-11 00:50:08,965 _internal.py[line:88] [process:30993] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/rpc/user/user_client.py', reloading
2018-07-11 00:56:02,404 logger.py[line:23] [process:32009] MainThread INFO init log
2018-07-11 00:56:02,643 _internal.py[line:88] [process:32022] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 00:56:02,688 _internal.py[line:88] [process:32022] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 00:56:04,491 logger.py[line:23] [process:32025] MainThread INFO init log
2018-07-11 00:56:04,726 _internal.py[line:88] [process:32043] MainThread WARNING  * Debugger is active!
2018-07-11 00:56:04,825 _internal.py[line:88] [process:32043] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 01:13:06,765 logger.py[line:23] [process:35738] MainThread INFO init log
2018-07-11 01:13:07,080 _internal.py[line:88] [process:35751] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 01:13:07,154 _internal.py[line:88] [process:35751] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 01:13:08,943 logger.py[line:23] [process:35756] MainThread INFO init log
2018-07-11 01:13:09,332 _internal.py[line:88] [process:35770] MainThread WARNING  * Debugger is active!
2018-07-11 01:13:09,398 _internal.py[line:88] [process:35770] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 01:13:27,113 _internal.py[line:88] [process:35770] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 01:13:27] "GET /user HTTP/1.1" 200 -
2018-07-11 01:13:38,715 _internal.py[line:88] [process:35770] Thread-22 INFO 127.0.0.1 - - [11/Jul/2018 01:13:38] "GET /notebook HTTP/1.1" 200 -
2018-07-11 01:13:51,854 _internal.py[line:88] [process:35770] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/api_kuaibiji/app/views/index.py', reloading
2018-07-11 01:13:51,857 _internal.py[line:88] [process:35770] Thread-13 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/api_kuaibiji/app/views/index.py', reloading
2018-07-11 01:13:59,921 logger.py[line:23] [process:35901] MainThread INFO init log
2018-07-11 01:14:00,225 _internal.py[line:88] [process:35917] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 01:14:00,296 _internal.py[line:88] [process:35917] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 01:14:02,159 logger.py[line:23] [process:35920] MainThread INFO init log
2018-07-11 01:14:02,459 _internal.py[line:88] [process:35935] MainThread WARNING  * Debugger is active!
2018-07-11 01:14:02,502 _internal.py[line:88] [process:35935] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 01:14:05,750 _internal.py[line:88] [process:35935] Thread-16 INFO 127.0.0.1 - - [11/Jul/2018 01:14:05] "GET /notebook HTTP/1.1" 200 -
2018-07-11 01:14:12,842 logger.py[line:23] [process:35963] MainThread INFO init log
2018-07-11 01:14:13,147 _internal.py[line:88] [process:35979] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 01:14:13,210 _internal.py[line:88] [process:35979] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 01:14:15,003 logger.py[line:23] [process:35982] MainThread INFO init log
2018-07-11 01:14:15,282 _internal.py[line:88] [process:35997] MainThread WARNING  * Debugger is active!
2018-07-11 01:14:15,313 _internal.py[line:88] [process:35997] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 01:14:23,180 _internal.py[line:88] [process:35997] Thread-14 INFO 127.0.0.1 - - [11/Jul/2018 01:14:23] "GET /notebook HTTP/1.1" 200 -
2018-07-11 01:26:04,527 logger.py[line:23] [process:37437] MainThread INFO init log
2018-07-11 01:26:04,750 _internal.py[line:88] [process:37457] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 01:26:04,792 _internal.py[line:88] [process:37457] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 01:26:06,546 logger.py[line:23] [process:37463] MainThread INFO init log
2018-07-11 01:26:06,775 _internal.py[line:88] [process:37476] MainThread WARNING  * Debugger is active!
2018-07-11 01:26:06,809 _internal.py[line:88] [process:37476] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 01:26:21,983 _internal.py[line:88] [process:37476] Thread-20 INFO 127.0.0.1 - - [11/Jul/2018 01:26:21] "GET /notebook HTTP/1.1" 200 -
2018-07-11 01:26:32,543 _internal.py[line:88] [process:37476] Thread-12 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/api_kuaibiji/service.py', reloading
2018-07-11 01:26:32,545 _internal.py[line:88] [process:37476] Thread-12 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/micro_services/api_kuaibiji/service.py', reloading
2018-07-11 01:53:34,598 logger.py[line:23] [process:40462] MainThread INFO init log
2018-07-11 01:53:34,803 _internal.py[line:88] [process:40473] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 01:53:34,848 _internal.py[line:88] [process:40473] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 01:53:36,500 logger.py[line:23] [process:40479] MainThread INFO init log
2018-07-11 01:53:36,700 _internal.py[line:88] [process:40490] MainThread WARNING  * Debugger is active!
2018-07-11 01:53:36,738 _internal.py[line:88] [process:40490] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 01:56:21,004 _internal.py[line:88] [process:40490] Thread-12 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/test/main_test.py', reloading
2018-07-11 01:56:21,007 _internal.py[line:88] [process:40490] Thread-12 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/test/main_test.py', reloading
2018-07-11 13:38:49,264 logger.py[line:23] [process:60298] MainThread INFO init log
2018-07-11 13:38:49,398 _internal.py[line:88] [process:60303] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 13:38:49,404 _internal.py[line:88] [process:60303] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 13:38:50,534 logger.py[line:23] [process:60306] MainThread INFO init log
2018-07-11 13:38:50,623 _internal.py[line:88] [process:60311] MainThread WARNING  * Debugger is active!
2018-07-11 13:38:50,658 _internal.py[line:88] [process:60311] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 13:46:54,254 logger.py[line:23] [process:60704] MainThread INFO init log
2018-07-11 13:49:11,863 logger.py[line:23] [process:60815] MainThread INFO init log
2018-07-11 13:49:16,339 logger.py[line:23] [process:60860] MainThread INFO init log
2018-07-11 13:49:33,494 logger.py[line:23] [process:60901] MainThread INFO init log
2018-07-11 13:49:53,930 logger.py[line:23] [process:60934] MainThread INFO init log
2018-07-11 13:49:54,131 _internal.py[line:88] [process:60947] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 13:49:54,168 _internal.py[line:88] [process:60947] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 13:49:56,450 logger.py[line:23] [process:60950] MainThread INFO init log
2018-07-11 13:49:56,679 _internal.py[line:88] [process:60963] MainThread WARNING  * Debugger is active!
2018-07-11 13:49:56,712 _internal.py[line:88] [process:60963] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 13:49:56,898 _internal.py[line:88] [process:60963] Thread-14 INFO 127.0.0.1 - - [11/Jul/2018 13:49:56] "GET / HTTP/1.1" 200 -
2018-07-11 13:50:01,150 _internal.py[line:88] [process:60963] Thread-21 INFO 127.0.0.1 - - [11/Jul/2018 13:50:01] "GET /user HTTP/1.1" 404 -
2018-07-11 13:50:08,992 _internal.py[line:88] [process:60963] Thread-22 INFO 127.0.0.1 - - [11/Jul/2018 13:50:08] "GET /user/user_info HTTP/1.1" 200 -
2018-07-11 13:52:38,472 _internal.py[line:88] [process:60963] Thread-27 INFO 127.0.0.1 - - [11/Jul/2018 13:52:38] "GET /user/user_info HTTP/1.1" 200 -
2018-07-11 13:52:50,815 logger.py[line:23] [process:61106] MainThread INFO init log
2018-07-11 13:52:50,908 _internal.py[line:88] [process:61112] MainThread INFO  * Running on http://0.0.0.0:5002/ (Press CTRL+C to quit)
2018-07-11 13:52:50,917 _internal.py[line:88] [process:61112] MainThread INFO  * Restarting with fsevents reloader
2018-07-11 13:52:51,945 logger.py[line:23] [process:61115] MainThread INFO init log
2018-07-11 13:52:52,030 _internal.py[line:88] [process:61120] MainThread WARNING  * Debugger is active!
2018-07-11 13:52:52,052 _internal.py[line:88] [process:61120] MainThread INFO  * Debugger PIN: 202-833-239
2018-07-11 13:55:07,768 _internal.py[line:88] [process:61120] Thread-2 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/database/mongo_manger.py', reloading
2018-07-11 13:55:07,769 _internal.py[line:88] [process:61120] Thread-2 INFO  * Detected change in '/Users/Shared/dbj/api.kuaibi/micro_service/database/mongo_manger.py', reloading
2018-07-11 13:55:08,982 logger.py[line:23] [process:61200] MainThread INFO init log
2018-07-11 13:55:23,623 logger.py[line:23] [process:61220] MainThread INFO init log
2018-07-11 13:56:10,499 logger.py[line:23] [process:61267] MainThread INFO init log
2018-07-11 13:56:37,545 logger.py[line:23] [process:61298] MainThread INFO init log
2018-07-11 13:57:00,306 logger.py[line:23] [process:61313] MainThread INFO init log
2018-07-11 13:57:13,107 logger.py[line:23] [process:61330] MainThread INFO init log
2018-07-11 13:57:21,543 logger.py[line:23] [process:61348] MainThread INFO init log
2018-07-11 13:57:42,388 logger.py[line:23] [process:61370] MainThread INFO init log
2018-07-11 13:58:08,436 logger.py[line:23] [process:61402] MainThread INFO init log
2018-07-11 13:58:16,480 logger.py[line:23] [process:61417] MainThread INFO init log
2018-07-11 13:58:25,302 logger.py[line:23] [process:61438] MainThread INFO init log
2018-07-11 14:06:54,527 logger.py[line:23] [process:61712] MainThread INFO init log
2018-07-11 14:07:48,082 logger.py[line:23] [process:61800] MainThread INFO init log
2018-07-11 14:08:28,906 logger.py[line:23] [process:61823] MainThread INFO init log
2018-07-11 14:08:57,028 logger.py[line:23] [process:61856] MainThread INFO init log
2018-07-11 14:09:25,339 logger.py[line:23] [process:61892] MainThread INFO init log
2018-07-11 14:09:31,898 logger.py[line:23] [process:61909] MainThread INFO init log
2018-07-11 14:10:35,083 logger.py[line:23] [process:61965] MainThread INFO init log
2018-07-11 14:10:53,724 logger.py[line:23] [process:61992] MainThread INFO init log
2018-07-11 14:11:00,111 logger.py[line:23] [process:62005] MainThread INFO init log
2018-07-11 14:15:30,709 logger.py[line:23] [process:62154] MainThread INFO init log
2018-07-11 14:17:22,015 logger.py[line:23] [process:62235] MainThread INFO init log
2018-07-11 14:17:26,322 logger.py[line:23] [process:62249] MainThread INFO init log
2018-07-11 14:18:01,482 logger.py[line:23] [process:62287] MainThread INFO init log
2018-07-11 14:18:22,659 logger.py[line:23] [process:62303] MainThread INFO init log
2018-07-11 14:19:04,245 logger.py[line:23] [process:62347] MainThread INFO init log
2018-07-11 14:20:44,874 logger.py[line:23] [process:62435] MainThread INFO init log
2018-07-11 14:20:59,465 logger.py[line:23] [process:62468] MainThread INFO init log
2018-07-11 14:21:18,464 logger.py[line:23] [process:62501] MainThread INFO init log
2018-07-11 14:22:45,788 logger.py[line:23] [process:62594] MainThread INFO init log
2018-07-11 14:23:04,784 logger.py[line:23] [process:62628] MainThread INFO init log
2018-07-11 14:23:07,355 logger.py[line:23] [process:62641] MainThread INFO init log
2018-07-11 14:26:15,296 logger.py[line:23] [process:62773] MainThread INFO init log
2018-07-11 14:26:55,675 logger.py[line:23] [process:62818] MainThread INFO init log
2018-07-11 14:26:58,404 logger.py[line:23] [process:62832] MainThread INFO init log
2018-07-11 14:29:14,108 logger.py[line:23] [process:62941] MainThread INFO init log
2018-07-11 14:33:07,487 logger.py[line:23] [process:63234] MainThread INFO init log
2018-07-11 14:33:11,215 logger.py[line:23] [process:63246] MainThread INFO init log
2018-07-11 14:33:33,323 logger.py[line:23] [process:63261] MainThread INFO init log
2018-07-11 14:34:13,299 logger.py[line:23] [process:63320] MainThread INFO init log
2018-07-11 14:34:55,172 logger.py[line:23] [process:63384] MainThread INFO init log
2018-07-11 14:35:02,750 logger.py[line:23] [process:63397] MainThread INFO init log
2018-07-11 14:35:24,737 logger.py[line:23] [process:63411] MainThread INFO init log
2018-07-11 14:35:41,559 logger.py[line:23] [process:63424] MainThread INFO init log
2018-07-11 14:37:25,008 logger.py[line:23] [process:63490] MainThread INFO init log
2018-07-11 14:53:24,813 logger.py[line:23] [process:63884] MainThread INFO init log
2018-07-11 14:53:54,618 logger.py[line:23] [process:64135] MainThread INFO init log
2018-07-11 14:55:33,289 logger.py[line:23] [process:64237] MainThread INFO init log
2018-07-11 14:55:36,817 logger.py[line:23] [process:64246] MainThread INFO init log
2018-07-11 14:55:52,307 logger.py[line:23] [process:64276] MainThread INFO init log
2018-07-11 14:56:00,669 logger.py[line:23] [process:64301] MainThread INFO init log
2018-07-11 14:56:38,736 logger.py[line:23] [process:64401] MainThread INFO init log
2018-07-11 14:56:54,468 logger.py[line:23] [process:64422] MainThread INFO init log
2018-07-11 14:57:28,084 logger.py[line:23] [process:64467] MainThread INFO init log
2018-07-11 14:57:33,189 logger.py[line:23] [process:64477] MainThread INFO init log
2018-07-11 14:58:02,248 logger.py[line:23] [process:64519] MainThread INFO init log
2018-07-11 14:58:12,688 logger.py[line:23] [process:64547] MainThread INFO init log
2018-07-11 15:12:07,860 logger.py[line:23] [process:65923] MainThread INFO init log
2018-07-11 15:12:52,006 logger.py[line:23] [process:65987] MainThread INFO init log
2018-07-11 15:12:55,613 logger.py[line:23] [process:66005] MainThread INFO init log
2018-07-11 15:13:20,794 logger.py[line:23] [process:66074] MainThread INFO init log
2018-07-11 16:07:23,370 logger.py[line:23] [process:71254] MainThread INFO init log
2018-07-11 16:08:32,185 logger.py[line:23] [process:71399] MainThread INFO init log
2018-07-11 16:08:37,409 logger.py[line:23] [process:71413] MainThread INFO init log
2018-07-11 16:09:04,205 logger.py[line:23] [process:71473] MainThread INFO init log
2018-07-11 16:10:12,235 logger.py[line:23] [process:71588] MainThread INFO init log
2018-07-11 16:10:14,681 logger.py[line:23] [process:71603] MainThread INFO init log
2018-07-11 17:00:48,239 logger.py[line:23] [process:75741] MainThread INFO init log
2018-07-11 17:00:56,111 logger.py[line:23] [process:75769] MainThread INFO init log
2018-07-11 17:02:34,749 logger.py[line:23] [process:75946] MainThread INFO init log
2018-07-11 17:32:43,052 logger.py[line:23] [process:78923] MainThread INFO init log
2018-07-11 17:36:00,349 logger.py[line:23] [process:79187] MainThread INFO init log
2018-07-11 17:44:24,332 logger.py[line:23] [process:79768] MainThread INFO init log
2018-07-11 17:46:16,539 logger.py[line:23] [process:79942] MainThread INFO init log
2018-07-11 17:46:34,753 logger.py[line:23] [process:79972] MainThread INFO init log
2018-07-11 17:46:41,623 logger.py[line:23] [process:79996] MainThread INFO init log
2018-07-11 17:47:18,869 logger.py[line:23] [process:80066] MainThread INFO init log
2018-07-11 17:48:23,009 logger.py[line:23] [process:80183] MainThread INFO init log
2018-07-11 17:48:46,059 logger.py[line:23] [process:80226] MainThread INFO init log
2018-07-11 17:54:04,967 logger.py[line:23] [process:80880] MainThread INFO init log
2018-07-11 17:55:35,084 logger.py[line:23] [process:81118] MainThread INFO init log
2018-07-11 17:55:49,037 logger.py[line:23] [process:81168] MainThread INFO init log
2018-07-11 17:56:34,123 logger.py[line:23] [process:81316] MainThread INFO init log
2018-07-11 17:56:40,112 logger.py[line:23] [process:81341] MainThread INFO init log
2018-07-11 17:57:13,598 logger.py[line:23] [process:81438] MainThread INFO init log
2018-07-11 17:58:05,271 logger.py[line:23] [process:81569] MainThread INFO init log
2018-07-11 18:03:15,325 logger.py[line:23] [process:82170] MainThread INFO init log
2018-07-11 18:17:05,011 logger.py[line:23] [process:83245] MainThread INFO init log
2018-07-11 18:17:37,170 logger.py[line:23] [process:83313] MainThread INFO init log
2018-07-11 18:17:49,995 logger.py[line:23] [process:83345] MainThread INFO init log
2018-07-11 18:18:14,593 logger.py[line:23] [process:83395] MainThread INFO init log
2018-07-11 18:18:17,279 logger.py[line:23] [process:83404] MainThread INFO init log
2018-07-11 18:19:55,181 logger.py[line:23] [process:83537] MainThread INFO init log
2018-07-11 18:21:14,731 logger.py[line:23] [process:83648] MainThread INFO init log
2018-07-11 18:21:26,506 logger.py[line:23] [process:83680] MainThread INFO init log
2018-07-11 18:21:35,259 logger.py[line:23] [process:83702] MainThread INFO init log
2018-07-11 18:24:08,341 logger.py[line:23] [process:84007] MainThread INFO init log
2018-07-11 18:35:22,068 logger.py[line:23] [process:84536] MainThread INFO init log
2018-07-11 19:43:08,405 logger.py[line:23] [process:86410] MainThread INFO init log
2018-07-11 19:44:21,463 logger.py[line:23] [process:86630] MainThread INFO init log
2018-07-11 19:44:40,973 logger.py[line:23] [process:86672] MainThread INFO init log
2018-07-11 19:49:00,675 logger.py[line:23] [process:87149] MainThread INFO init log
2018-07-11 20:05:05,250 logger.py[line:23] [process:87850] MainThread INFO init log
2018-07-11 20:16:06,628 logger.py[line:23] [process:88420] MainThread INFO init log
